internal/view/table.go

```diff
@@ -208,8 +207,12 @@ func (t *Table) cpCmd(evt *tcell.EventKey) *tcell.EventKey {
 		return evt
 	}
 	_, n := client.Namespaced(path)
-	v := tview.NewTextView().SetText(n)
-	cpCmd(t.app.Flash(), v)(evt)
+	err := clipboardWrite(n)
```

Comment:
No need for the else here. If there is an error you should just return

Reply:
Thank you Takumi for making these corrections!! I'll update the final comment and fix the build as I am getting ready for a new drop and would like to include your PR.